package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@babel/plugin-proposal-decorators": "^7.8.3",
     "@rails/actioncable": "^6.0.0",
```

Comment:
Better to upgrade all rails dependencies to version 6.1 

"@rails/actioncable": "^6.0.0",
 "@rails/activestorage": "^6.0.0",
 "@rails/ujs": "^6.0.0",

Reply:
out of scope, I guess. Let's make it when we will close vulnerabilities listed by github (see https://github.com/helphubteam/helphub/security/dependabot)